pynest/nest/lib/hl_api_spatial.py

```diff
@@ -728,6 +731,81 @@ def GetTargetNodes(sources, tgt_layer, syn_model=None):
     return tuple(src_tgt_map[snode_id] for snode_id in sources.tolist())
 
 
+def GetSourceNodes(src_layer, targets, syn_model=None):
+    """
+    Obtain sources of `targets` in given `source` population.
+
+    For each neuron in `targets`, this function finds all target elements
+    in `src_layer`. If `syn_model` is not given (default), all sources are
+    returned, otherwise only targets of specific type.
+
+    Parameters
+    ----------
+    src_layer : NodeCollection
+        NodeCollection with node IDs of `src_layer`
+    targets : NodeCollection
+        NodeCollection with node IDs of `targets`
+    syn_model : [None | str], optional, default: None
+        Return only target positions for a given synapse model.
+
+    Returns
+    -------
+    tuple of NodeCollection:
+        Tuple of `NodeCollections` of source neurons fulfilling the given criteria, one `NodeCollection` per
+        target node ID in `target`.
+
+    See also
+    --------
+    GetSourcePositions: Obtain positions of sources in a given source layer connected to given target.
+    GetConnections: Return connection identifiers between
+        sources and targets
```

Comment:
Should "Return connection identifiers between" and "sources and targets" be on the same line here?

Reply:
They should, thanks. Corrected in commit 7617304